### PR TITLE
Work in progress

### PR DESCRIPTION
**Pull request by embryCODE**
I'll leave this PR open for comments

### CODE REVIEW

File: src/components/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            width: '100%',
+            margin: '0 auto',
+            position: 'absolute',
+          }}
+        />
+      </Box>
+      <Image
+        fluid={data.bandShot.childImageSharp.fluid}
+        alt="A photo of The Downtown Band"
+        style={{ width: '100%', maxWidth: 1200, margin: '0 auto' }}
```
Comment (cjharkins) [inline src/components/Hero.tsx]: pull in config for max width

File: src/components/Hero.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  return (
+    <Box
+      sx={{
+        background: `url(${heroBg}) no-repeat center`,
+        backgroundSize: 'cover',
+      }}
+    >
+      <Box
+        sx={{
+          maxWidth: 1200,
```
Comment (cjharkins) [inline src/components/Hero.tsx]: Aren't you setting the maxWidth in the config.ts? pull in config so you don't get hung up if needed to change.

File: src/pages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        <YouTube id="https://www.youtube.com/embed/0WzacLIkK2c" />
+
+        <Box>
+          {/* TODO: Provide links to these claims */}
+          <Text as="p">
+            Voted{' '}
+            <Link href="https://google.com">
+              &ldquo;Best Live Band in Tennessee&rdquo;
+            </Link>{' '}
+            &{' '}
```
Comment (cjharkins) [inline src/pages/index.tsx:35]: Trivial at best, can't you just do `{ ' & ' }` ?

Comment (embryCODE) [reply src/pages/index.tsx:35]: Yeah, for sure. That's just WebStorm's automatic handling of spaces. I didn't even look at it.

File: src/components/Hero.tsx (hunk 2 above)
Comment (embryCODE) [reply src/components/Hero.tsx]: Yup, good catch

**Pull request closed without merging**